### ISSUE TITLE
Stats: Crash when opening the View all Followers

### ISSUE THREAD

**Issue by 0nko** [wordpress-mobile/WordPress-Android#9511]
Stack trace:
```
Fatal Exception: kotlin.UninitializedPropertyAccessException: lateinit property siteModel has not been initialized
       at org.wordpress.android.ui.stats.refresh.utils.StatsSiteProvider.getSiteModel(StatsSiteProvider.java:17)
       at org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases.FollowersUseCase.loadCachedData(FollowersUseCase.java:65)
       at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase.fetch(BaseStatsUseCase.java:78)
       at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$fetch$1.invokeSuspend(BaseStatsUseCase.java)
       at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(BaseContinuationImpl.java:32)
       at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.java:233)
       at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.java:594)
       at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.java:60)
       at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.java:742)
```

**Comment by designsimply** [comment]
<sub>(internal reference: p1554313653074300-slack-mobile-support)</sub>

**Comment by jtreanor** [comment]
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9512. The fix is included in `12.1-rc-4`/`alpha-161`.
